subprojects/analysis-kotlin-api/src/test/kotlin/org/jetbrains/dokka/analysis/test/ExternalDocumentableProviderTest.kt

```diff
@@ -0,0 +1,293 @@
+/*
+ * Copyright 2014-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package org.jetbrains.dokka.analysis.test
+
+import org.jetbrains.dokka.analysis.test.api.javaTestProject
+import org.jetbrains.dokka.analysis.test.api.kotlinJvmTestProject
+import org.jetbrains.dokka.analysis.test.api.useServices
+import org.jetbrains.dokka.analysis.test.api.util.getResourceAbsolutePath
+import org.jetbrains.dokka.links.DRI
+import org.jetbrains.dokka.model.*
+import kotlin.test.*
+
+class ExternalDocumentableProviderTest {
```

Comment:
Is this yet another set of tests for external documentable? 
What are the plans for `ExternalDocumentablesTest`? It also contains interesting corner cases.

Reply:
Deleted the old tests, as the new tests should cover all sensible corner cases I could think of